src/app/zap-templates/templates/app/cluster-enums.zapt

```diff
@@ -28,7 +28,7 @@ k{{asUpperCamelCase label}} = {{asHex value 2}},
 {{#if (isWeaklyTypedEnum label)}}
 #else // CHIP_USE_ENUM_CLASS_FOR_IM_ENUM
 using {{asType label}} = EmberAf{{asType label}};
-#endif
+#endif // CHIP_USE_ENUM_CLASS_FOR_IM_ENUM
 {{/if}}
```

Comment:
Don't we need to output a kInvalid or something for the `first_unused_enum_value` bit?  How are cluster impls supposed to know that the value is invalid otherwise?

Reply:
just generated this kvalue, thanks